SQL_module/SubjectForm/db_manipulation.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+function executeQuery($sql)
+    {
+        $host = "localhost";
+        $dbname = "test";
+        $username = "root";
+        $pass = "";
+
+        try {
+            $connection = new PDO("mysql:host=$host;dbname=$dbname;charset=utf8", $username, $pass, array(PDO::MYSQL_ATTR_INIT_COMMAND => "SET NAMES utf8"));
+            $connection->query($sql);
+
+            $sql = "SELECT * FROM electives";
```

Comment:
I think it would be better if you have 2 separate functions - one for executing the SELECT and another executing the actual query that was provided. Currently executeQuery() is executing more than it is expected by its name and also has multiple responsibilities - it executes a provided query, then executes a second query and outputs the result of the second one. Please split the presentation logic and the execution of query logic. Currently it is quite confusing what exactly happens here.